apps/prairielearn/src/pages/studentInstanceQuestionHomework/studentInstanceQuestionHomework.ejs

```diff
@@ -49,7 +49,5 @@
         </div>
       </div>
     </main>
-    <!-- Needs to be located below element scripts -->
-    <script src="<%= asset_path('localscripts/questionPageScripts.js') %>"></script>
```

Comment:
Don't we still need the question scripts on this page?

Reply:
The functions have been moved to question.ts, though check the comment in the other thread.